README.md

```diff
@@ -1 +1,2 @@
-# Sample_Web
\ No newline at end of file
+# Sample_Web
+1. Commit - just the index.html file with HTML headings, tables and forms
```

Comment:
interesting you chose to do this. A lot of pro open source repos use a CHANGELOG.md file that also contains something like:
```
* 2020-09-06:
  * changed: N/A
  * added: some new HTML
* 2020-09-07:
  ...
```

Reply:
interesting. maybe you can elaborate on this on the next call.

Reply:
i mean i'm not asking you to add one haha. just fyi.

Check out the react project repo's changelog - they list the changes in each version so that you as a developer can keep track: https://github.com/facebook/react/blob/master/CHANGELOG.md